source/mod_dist.f90

```diff
@@ -20,45 +23,138 @@
+  logical,                  private, save :: dist_libRead   = .false. ! Read file with dist library instead of internal reader
+  integer,                  private, save :: dist_updtEFrom = 3       ! The parameter sent to part_updatePartEnergy after DIST
+  character(len=mFileName), private, save :: dist_distFile  = " "     ! File name for reading the distribution
+  character(len=mFileName), private, save :: dist_echoFile  = " "     ! File name for echoing the distribution
+
+  integer,          allocatable, private, save :: dist_colFormat(:)   ! The column types in the FORMAT
+  real(kind=fPrec), allocatable, private, save :: dist_colScale(:)    ! Scaling factor for the columns
+  integer,                       private, save :: dist_nColumns  = 0  ! The number of columns in the FORMAT
+
+  character(len=:), allocatable, private, save :: dist_partLine(:)    ! PARTICLE definitions in the block
+  integer,                       private, save :: dist_nParticle = 0  ! Number of PARTICLE keywords in block
+
+  integer,                       private, save :: dist_numPart   = 0  ! Number of actual particles generated or read
+
+  !  Column formats
+  ! ================
+  integer, parameter :: dist_fmtNONE        = 0  ! Column ignored
+  integer, parameter :: dist_fmtPartID      = 1  ! Paricle ID
+  integer, parameter :: dist_fmtParentID    = 2  ! Particle parent ID (for secondary particles, otherwise equal particle ID)
+
+  ! Physical Coordinates
+  integer, parameter :: dist_fmtX           = 11 ! Horizontal position
+  integer, parameter :: dist_fmtY           = 12 ! Vertical positiom
+  integer, parameter :: dist_fmtXP          = 13 ! Horizontal angle
+  integer, parameter :: dist_fmtYP          = 14 ! Vertical angle
+  integer, parameter :: dist_fmtPX          = 15 ! Horizontal momentum
+  integer, parameter :: dist_fmtPY          = 16 ! Vertical momentum
+  integer, parameter :: dist_fmtPXP0        = 17 ! Relative horizontal momentum
+  integer, parameter :: dist_fmtPYP0        = 18 ! Relative vertical momentum
+  integer, parameter :: dist_fmtSIGMA       = 19 ! Longitudinal relative position
+  integer, parameter :: dist_fmtDT          = 20 ! Time delay
+  integer, parameter :: dist_fmtE           = 21 ! Particle energy
+  integer, parameter :: dist_fmtP           = 22 ! Particle momentum
+  integer, parameter :: dist_fmtDEE0        = 23 ! Relative particle energy (to reference particle)
+  integer, parameter :: dist_fmtDPP0        = 24 ! Relative particle momentum (to reference particle)
+
+  ! Normalised Coordinates
+  integer, parameter :: dist_fmtX_NORM      = 31 ! Normalised horizontal position
+  integer, parameter :: dist_fmtY_NORM      = 32 ! Normalised vertical positiom
+  integer, parameter :: dist_fmtXP_NORM     = 33 ! Normalised horizontal angle
+  integer, parameter :: dist_fmtYP_NORM     = 34 ! Normalised vertical angle
+  integer, parameter :: dist_fmtSIGMA_NORM  = 35 ! Normalised longitudinal relative position
```

Comment:
I would call
`dist_fmtZETA_NORM  = 35 ! Normalised longitudinal relative position`

Reply:
Will add this as well.